entity-services/src/main/xdmp/entity-services/entity-services-codegen.xqy

```diff
@@ -570,6 +570,7 @@ declare function {$module-prefix}:convert-instance-{$entity-type-name}(
         if (exists($source-entity-type))
         then
             for $property-name in map:keys(map:get($source-entity-type, "properties"))
+            where not($property-name = map:keys(map:get($entity-type, "properties")))
```

Comment:
@grechaw While this takes care of overreporting missing properties, the return statement after else, returns the comment "(: The following properties are in the source, but not the target:  :)"

```
declare function test-8-from-test-8:convert-instance-Customer(
    $source-node as node()
) as map:map
{
    json:object()
    (: The following line identifies the type of this instance.  Do not change it. :)
    =>map:with('$type', 'Customer')
    (: The following lines are generated from the 'Customer' entity type. :)
    =>   map:with('CustomerId',             xs:integer($source-node/Customer/CustomerId))
    =>es:optional('CompanyName',            xs:string($source-node/Customer/CompanyName))
    =>es:optional('Country',                xs:string($source-node/Customer/Country))
    =>es:optional('ContactName',            xs:string($source-node/Customer/ContactName))
    (: The following properties are in the source, but not the target: 
  :)

};
```